dbr_test.go

```diff
@@ -58,14 +61,21 @@ type nullTypedRecord struct {
 }
 
 func reset(t *testing.T, sess *Session) {
-	var autoIncrementType string
+	autoIncrementType := ""
+	boolType := "bool"
+	datetimeType := "timestamp"
+
 	switch sess.Dialect {
 	case dialect.MySQL:
 		autoIncrementType = "serial PRIMARY KEY"
 	case dialect.PostgreSQL:
 		autoIncrementType = "serial PRIMARY KEY"
```

Comment:
Everything ok, but if you chose to use default values, you could also make `autoIncrementType="serial PRIMARY KEY"`, just to be consistent. I wouldn't mind about that, but you'll need to update PR anyways.

Reply:
fixed